<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitoringSystemWindow</class>
 <widget class="QMainWindow" name="MonitoringSystemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(64, 65, 66);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="start_page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QPushButton" name="start_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>330</width>
            <height>330</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(64, 65, 66);
	background-color: rgb(190, 192, 193);
	border-radius: 165px;
    border-style: outset;
}</string>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="start_midlow_verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="start_left_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="start_midup_verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="start_right_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="main_page">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border: none;
}

QListWidget {
	background-color: rgb(46, 47, 48);
	color: rgb(208, 208, 208);
	border: none;
}

QListWidget::item {
	border: none;
}

QListWidget::item:hover {
	background-color: rgb(73, 74, 75);
}

QListWidget::item:selected {
	background-color: rgb(73, 74, 75);
	color: rgb(208, 208, 208);
}

QSplitter::handle {
	background: rgb(6, 8, 10);
}

</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="agents_to_logs_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="handleWidth">
              <number>1</number>
             </property>
             <property name="childrenCollapsible">
              <bool>false</bool>
             </property>
             <widget class="QSplitter" name="agents_to_info_splitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="handleWidth">
               <number>1</number>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="QListWidget" name="agents_listWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Loaded agents list.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="toolTipDuration">
                <number>5000</number>
               </property>
               <property name="whatsThis">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
               </property>
               <property name="tabKeyNavigation">
                <bool>true</bool>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
               <property name="movement">
                <enum>QListView::Static</enum>
               </property>
               <property name="flow">
                <enum>QListView::TopToBottom</enum>
               </property>
               <property name="isWrapping" stdset="0">
                <bool>false</bool>
               </property>
               <property name="resizeMode">
                <enum>QListView::Adjust</enum>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
               <property name="uniformItemSizes">
                <bool>false</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="selectionRectVisible">
                <bool>true</bool>
               </property>
               <property name="itemAlignment">
                <set>Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QFrame" name="agents_info_frame">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QFrame {
	background-color: rgb(46, 47, 48);
}

QLabel {
	color: rgb(208, 208, 208);
}

QComboBox QAbstractItemView {
        background-color: white;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="QSplitter" name="logs_to_errs_splitter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="handleWidth">
               <number>1</number>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="QListWidget" name="logs_listWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
              <widget class="QSplitter" name="errors_to_criticals_splitter">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="handleWidth">
                <number>1</number>
               </property>
               <property name="childrenCollapsible">
                <bool>false</bool>
               </property>
               <widget class="QListWidget" name="errors_listWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::ActionsContextMenu</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
               <widget class="QListWidget" name="criticals_listWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::ActionsContextMenu</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
